WorkflowFunctionAdminBundle/DataFixtures/MongoDB/LoadGroupDemoData.php

```diff
@@ -19,6 +19,7 @@ class LoadGroupDemoData extends AbstractLoadGroupData implements OrchestraProduc
     public function load(ObjectManager $manager)
     {
         $this->addRole('group2');
+        $this->addRole('group-admin');
```

Comment:
Yes, I have three PR related one to another. two of theme are validated this is the last one.